jekyll/_cci2/roles-and-permissions-overview.adoc

```diff
@@ -0,0 +1,443 @@
+^|
+^|
+
+| *Triggers*
+|
+|
+|
+
+^| View triggers
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| Trigger build
+^| icon:check-circle[]
+^| icon:check-circle[]
+^|
+
+^| Edit triggers
+^| icon:check-circle[]
+^|
+^|
+
+| *Config sources*
+|
+|
+|
+
+^| View config sources
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| Edit config sources
+^| icon:check-circle[]
+^|
+^|
+
+|===
+
+
+[#project-role-permissions-matrix]
+=== Project role permissions matrix
+
+The table below shows the permissions associated with each CircleCI project role:
+
+[.table.table-striped]
+[cols=4*, options="header"]
+|===
+| ACTIONS
+
+3+^| PROJECT ROLES
+
+|
+| *Admin*
+| *Contributor*
+| *Viewer*
+
+| *Projects*
+|
+|
+|
+
+^| View projects
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| View project access
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| View project credentials
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| Manage project
+^| icon:check-circle[]
+^|
+^|
+
+| *Webhooks*
+|
+|
+|
+
+^| View project webhooks
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| Manage project webhooks
+^| icon:check-circle[]
+^|
+^|
+
+| *Schedule*
+|
+|
+|
+
+^| View schedule
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| Edit schedule
+^| icon:check-circle[]
+^|
+^|
+
+| *Triggers*
+|
+|
+|
+
+^| View triggers
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| Trigger build
+^| icon:check-circle[]
+^| icon:check-circle[]
+^|
+
+^| Edit triggers
+^| icon:check-circle[]
+^|
+^|
+
+| *Contexts*
+|
+|
+|
+
+^| View contexts
+^| icon:check-circle[]
+^| icon:check-circle[]
+^|
+
+^| Use contexts
+^| icon:check-circle[]
+^| icon:check-circle[]
+^|
+
+^| Edit context variables
+^|
+^|
+^|
+
+^| Manage contexts
+^|
+^|
+^|
+
+| *Config sources*
+|
+|
+|
+
+^| View config sources
+^| icon:check-circle[]
+^| icon:check-circle[]
+^| icon:check-circle[]
+
+^| Edit config sources
+^| icon:check-circle[]
+^|
+^|
+
+|===
+
+[#permissions-scope]
+== Permissions scope
+
+Your CircleCI roles and associated permissions are **not** derived from the permissions set in your version control system (VCS). Your CircleCI role permissions do **not** allow you to bypass permissions in the VCS.
+
+For example, you may be an _Organization Administrator_ within CircleCI, which gives you access to view and modify organization and project settings _within your CircleCI organization_. However, you will not be able to edit a project’s `.circleci/config.yml` hosted in your VCS without your user also having the write permissions _within that VCS's repository project_. Your CircleCI user’s VCS permissions are determined by its associated VCS identity.
+
+At this time, your VCS identity can be managed through your CircleCI connection when managing triggers and configurations.
```

Comment:
Not sure how to do this or what "VCS identity can be managed" means here exactly.